libraries/c_sdk/standard/mqtt/test/unit/iot_tests_mqtt_receive.c

```diff
@@ -569,7 +611,7 @@ TEST_SETUP( MQTT_Unit_Receive )
                                                                     1 ) );
 
     /* Clear functions called flags. */
-    _deserializeOverrideCalled = false;
+    _deserializeOverrideCalled = true;
```

Comment:
is this change required? If so, can you please add a comment about why this would be required

Reply:
Addressed